python/BioSimSpace/FreeEnergy/_ddg.py

```diff
@@ -0,0 +1,495 @@
+######################################################################
+# BioSimSpace: Making biomolecular simulation a breeze!
+#
+# Copyright: 2017-2023
+#
+# Authors: Lester Hedges <lester.hedges@gmail.com>, Matthew Burman <matthew@openbiosim.org>
+#
+# BioSimSpace is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# BioSimSpace is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with BioSimSpace. If not, see <http://www.gnu.org/licenses/>.
+######################################################################
+
+
+# Alchemical transfer analysis methods. UWHAM implementation adapted from
+# both the `femto` and `AToM-openmm` packages.
+__all__ = ["analyse_UWHAM", "analyse_MBAR", "new_MBAR"]
+
+import pandas as _pd
+
+import numpy
+import scipy.optimize
+import scipy.special
+import pathlib
+import functools
+import pathlib as _pathlib
+import os as _os
+import pymbar
```

Comment:
Can these all be private.

Reply:
Now private (https://github.com/OpenBioSim/biosimspace/blob/feature_AToM/python/BioSimSpace/FreeEnergy/_ddg.py#L27-L35)